code/modules/mob/living/carbon/xenomorph/mutators/strains/drone/healer.dm

```diff
@@ -94,14 +96,70 @@
 		return
 
 	if(health <= 0)
-		to_chat(src, SPAN_WARNING("You have no health left to give!"))
+		to_chat(src, SPAN_WARNING("You do not have enough health to make regenerative resin!"))
 		return
 
 	adjustBruteLoss(amount * damage_taken_mod)
 	updatehealth()
 	new /datum/effects/heal_over_time(target_xeno, amount, 10, 2)
 	target_xeno.xeno_jitter(1 SECONDS)
 	target_xeno.flick_heal_overlay(10 SECONDS, "#00be6f")
-	to_chat(target_xeno, SPAN_XENOWARNING("\The [src] has transfered some of their health to you. You feel reinvigorated!"))
-	to_chat(src, SPAN_XENOWARNING("You have transferred some of your health to \the [target_xeno]. You feel weakened..."))
+	to_chat(target_xeno, SPAN_XENOWARNING("\The [src] covers your wounds with regenerative resin. You feel reinvigorated!"))
+	to_chat(src, SPAN_XENOWARNING("You regurgitate your vital fluids to create a regenerative resin and apply it to \the [target_xeno]'s wounds. You feel weakened...")) //The vital fluids mix with sticky saliva.
 	playsound(src, "alien_drool", 25)
+	var/datum/behavior_delegate/drone_healer/healer_delegate = src.behavior_delegate
+	healer_delegate.salve_applied_recently = TRUE
```

Comment:
sanity check pls

Reply:
why do we need a sanity check here, all that would cause is a runtime not showing up, the bug will still be there